maliput_sim/simulation.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, road_network, sim_config: SimulationConfig):
 
     def add_agent(self, initial_state: AgentInitialState,
                   controller: Callable[[float, SimulationState, Entity, EntityComponentManager], None],
-                  get_state: Callable[[], dict]):
+                  get_state: Callable[[], dict] = lambda: {}):
```

Comment:
Is this default value actually a good choice?

Reply:
I thought it from the user perspective in case you don't want to obtain the state of your controller. However, it is true that the user in that case could simply provide an empty lambda and that's it. I don't have the use case for the moment to lean to a particular implementation.